ui/app/src/modules/Preview/ToolBar.tsx

```diff
@@ -226,17 +237,24 @@ export function AddressBar(props: AddressBarProps) {
             </MenuItem>
           ))}
         </Select>
-        <PagesSearchAhead
-          value={internalUrl}
-          placeholder={noSiteSet ? '' : '/'}
-          disabled={disabled}
-          onEnter={(value) => onUrlChange(value)}
-          classes={{
-            input: classes.input
-          }}
-          onFocus={() => setFocus(true)}
-          onBlur={() => setFocus(false)}
-        />
+        {!focus && item && (
+          <div className={classes.itemDisplayWrapper} onClick={() => setFocus(true)}>
+            <ItemDisplay item={item} showPreviewUrl={true} styles={{ root: { maxWidth: '100%' } }} />
```

Comment:
Adding previewUrl to the ItemDisplay is too wired. That's specific to this use case, just compose the components.
```suggestion
            <ItemDisplay item={item} styles={{ root: { maxWidth: '100%' } }} /> <Typography>{item.previewUrl}</Typography>
```